test/images/logs-generator/Dockerfile

```diff
@@ -1,22 +0,0 @@
-# Copyright 2016 The Kubernetes Authors.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-FROM BASEIMAGE
-
-ENV LOGS_GENERATOR_LINES_TOTAL 1
-ENV LOGS_GENERATOR_DURATION 1s
-
-COPY logs-generator /
-
-CMD ["sh", "-c", "/logs-generator --logtostderr --log-lines-total=${LOGS_GENERATOR_LINES_TOTAL} --run-duration=${LOGS_GENERATOR_DURATION}"]
```

Comment:
this explains my confusion: the old logs-generator container uses env vars to pass to the parameters; the agnhost refactor does not

Reply:
Yea the new one just uses flags. Based on https://github.com/kubernetes/kubernetes/pull/78394/files#diff-7dc1ebd3b2f2f6ca1ff248e6601016a6R108 it looks like the invocation was updated to use the flags instead of env vars too, so the doc notes about the env vars can be removed.

Reply:
Good catch, there's no longer a need for those env variables. I'll also remove them from the Dockerfile.